### DISCUSSION TITLE
.desktop file Ubuntu

### DISCUSSION THREAD

**Discussion by flaun** [jeffvli/sonixd#416] [General]
Hi Jeff,

I'm somehow to stupid to create a working .desktop file for sonixd in my ubuntu os (LTS 22.04). Would be cool, to have menu entry for Sonxd.
Could you please help me?

right now, it looks like this:
`[Desktop Entry]

Name=Sonixd
Comment=Music player
Exec="/usr/bin/sonixd" %U
Terminal=false
Type=Application
Icon=/opt/sonixd/resources/assets/icon.svg
StartupWMClass=sonixd
Categories=Audio;Player;`

what is wrong?

Thanks for great work. Love sonixd!
flaun

**Comment by WhyKickAmooCow** [comment]
If you are using the AppImage release, I would recommend installing https://github.com/TheAssassin/AppImageLauncher

After install this, when running the AppImage you should be prompted if you want to "integrate" the program and run it. When you do it will add it's own entry in your application menu.

**Comment by flaun** [comment]
Thanks for your quick aswer. I used the linux-x64.tar.xz image. So I guess I need to create a manual launcher.
Any other ideas?
flaun 

Am Sun, 15 Jan 2023 20:03:30 -0800, Adam schrieb:
> If you are using the AppImage release, I recommend 
> installing https://github.com/TheAssassin/AppImageLauncher
>
> After install this, when running the AppImage you should be 
> prompted if you want to "integrate" the program and run it. When 
> you do it will add it's own entry in your application menu.
>

-- 
Sent with Dekko - the convergent email app

**Comment by WhyKickAmooCow** [comment]
I don't see anything obviously wrong with the desktop file. You shouldn't need the quotes around the path to the sonixd executable in Exec though. Is the issue that you don't know where to put the desktop file to have it show up, or that it isn't launching sonixd?

**Comment by flaun** [comment]
I put the desktop file where all the others are. This shouldn't be the problem. Will try to have a look at it later this day. Thanks for ypur help.
flaun

Am Sun, 15 Jan 2023 21:29:32 -0800, Adam schrieb:
> I don't see anything obviously wrong with the desktop file. You 
> shouldn't need the quotes around the path to the sonixd 
> executable in Exec though. Is the issue that you don't know 
> where to put the desktop file to have it show up, or that it 
> isn't launching sonixd?
>

-- 
Sent with Dekko - the convergent email app
